overlays/holo-nixpkgs/holo-whitelist/holo-whitelist.py

```diff
@@ -0,0 +1,29 @@
+import json
+import requests
+import subprocess
+
+def main():
+    #temp creation of json
+    data = {
+            "email":"test@test.com"
+    }
+    with open('/var/lib/data.json', 'w') as outfile:
+        json.dump(data, outfile)
+    #end temp
+    path = '/var/lib/data.json'
+    datafile = open(path, 'r')    
+    loadedjson = json.load(datafile)
+    get_zt = subprocess.run(["zerotier-cli", "-j", "info"], capture_output=True)
+    #print(get_zt)
+    ztjson = json.loads(get_zt.stdout.decode('utf-8'))
```

Comment:
You actually don't need to decode it, `json.loads` will work on bytestrings as-is:

```suggestion
    ztjson = json.loads(get_zt.stdout))
```

Reply:
removed